demo/src/vendor.ts

```diff
@@ -0,0 +1,6 @@
+import 'angular2/bundles/angular2-polyfills';
```

Comment:
It just means that you don't see issue with your current use case. But there are problems. If I can't convince you maybe Misko will? https://github.com/angular/angular/issues/5304#issuecomment-158133227